src/cms/cms.js

```diff
@@ -1,4 +1,4 @@
-// tslint:disable-next-line:no-implicit-dependencies
-import CMS from 'netlify-cms'
-
-CMS.registerPreviewStyle('/cms.css')
+// Your module must at least include these three imports
+import React from "react";
+import CMS from "netlify-cms";
+import "netlify-cms/dist/cms.css";

```

Comment:
cms.css is only needed if we're generating custom previews (which may be good eventually, but currently not needed and is just wasting space in the code base)